### ISSUE TITLE
Startup error 

### ISSUE THREAD

**Issue by offerm** [ExchangeUnion/xud#462]
Getting this error after startup without a database.
```
admins-MacBook-Pro-2:resolver admin$ ./start_xud.bash 
9/12/2018, 8:43:55 AM [GLOBAL] info: config loaded
9/12/2018, 8:43:55 AM [GLOBAL] info: Local nodePubKey is 0235584e2ac9a34a13ce6f6dc9fb8aa1e7918d7701f0b4b69dc002887e6b3fbba3
9/12/2018, 8:43:56 AM [LND] info: LndClient status: DISCONNECTED
9/12/2018, 8:43:56 AM [LND] info: trying to verify connection to lnd with uri: localhost:10002
9/12/2018, 8:43:56 AM [LND] info: LndClient status: DISCONNECTED
9/12/2018, 8:43:56 AM [LND] info: trying to verify connection to lnd with uri: localhost:10001
(node:43461) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
9/12/2018, 8:43:56 AM [LND] info: LndClient status: CONNECTION_VERIFIED
9/12/2018, 8:43:56 AM [LND] info: LndClient status: CONNECTION_VERIFIED
9/12/2018, 8:43:56 AM [P2P] info: retrieved external IP: 84.111.139.147
9/12/2018, 8:43:56 AM [P2P] info: p2p server listening on 0.0.0.0:8885
9/12/2018, 8:43:56 AM [P2P] info: Connecting to known / previously connected peers
9/12/2018, 8:43:56 AM [P2P] debug: Verifying reachability of advertised address: 84.111.139.147:8885
9/12/2018, 8:43:56 AM [RPC] info: gRPC server listening on localhost:7001
9/12/2018, 8:43:56 AM [P2P] debug: Connected pre-handshake to peer xud2.test.exchangeunion.com:8885
9/12/2018, 8:43:56 AM [P2P] debug: Connected pre-handshake to peer xud1.test.exchangeunion.com:8885
9/12/2018, 8:43:56 AM [P2P] debug: Connected pre-handshake to peer xud3.test.exchangeunion.com:8885
9/12/2018, 8:43:56 AM [P2P] info: Completed start-up connections to known peers.
9/12/2018, 8:43:57 AM [P2P] verbose: received 0 orders from 028599d05b18c0c3f8028915a17d603416f7276c822b6b2d20e71a3502bd0f9e0a
9/12/2018, 8:43:57 AM [P2P] verbose: received 0 orders from 02b66438730d1fcdf4a4ae5d3d73e847a272f160fee2938e132b52cab0a0d9cfc6
9/12/2018, 8:43:57 AM [P2P] verbose: received 0 orders from 03fd337659e99e628d0487e4f87acf93e353db06f754dccc402f2de1b857a319d0
9/12/2018, 8:44:06 AM [P2P] warn: Could not verify reachability of advertised address: 84.111.139.147:8885
9/12/2018, 8:45:01 AM [P2P] info: Peer 02b66438730d1fcdf4a4ae5d3d73e847a272f160fee2938e132b52cab0a0d9cfc6 socket closed
events.js:167
      throw er; // Unhandled 'error' event
      ^

Error: connect ETIMEDOUT 84.111.139.147:8885
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1163:14)
Emitted 'error' event at:
    at emitErrorNT (internal/streams/destroy.js:82:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
    at process._tickCallback (internal/process/next_tick.js:63:19)
admins-MacBook-Pro-2:resolver admin$ 
```

**Comment by michael1011** [comment]
I don't have these problems neither on Node `v8.12.0` nor on `v10.10.0` on Linux. Could someone else try to reproduce this?

**Comment by sangaman** [comment]
I believe this is fixed by #452. I can merge it now or if you want to try pulling the changes down and verifying yourself let me know.
